### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by serviolee**
Hey guys, I'm rendering pictures and restaurant name from the database. Please review, thanks!

### CODE REVIEW

File: database/Populator.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const populateRestaurants = () => {
 
 const populateUsers = () => {
   for (let i = 0; i < 100; i++) {
-    let userNames = ['Neon', 'Omega', 'Kookaburra', 'RagingBull', 'Asylum', 'Flurry', 'Ollie', 'Scoobie', 'Brave', 'Guaymas', 'Pinochio', 'Danana', 'Pepita', 'Padila', 'Ninja', 'Bonza'];
+    let userNames = ['Ken Y.', 'Omega S.', 'Johnny J.', 'Bald B.', 'Bison M.', 'Guile T.', 'Ollie C.', 'Scoobie D.', 'Brave H.', 'Rye T.', 'Chun L.', 'Blanka B.', 'Zangief K.', 'Dhalsim B.', 'Sagat P.', 'Vega B.', 'Charlie T.', 'Michael W.', 'Servio L.', 'Trevor P.'];
     let randomIndex = randomNumberGenerator(0, userNames.length);
```
Comment (cthao309) [inline database/Populator.js:34]: Usually, you declared variables and or array outside of the loop so it doesn't create those variable every time it iterates through the for loop

File: client/src/components/Gallery.jsx [hunk 2]
```diff
@@ -27,6 +27,27 @@ class Gallery extends React.Component {
     this.handleSliderClick = this.handleSliderClick.bind(this);
   }
 
+  componentDidMount() {
+    axios
+    .get('/restaurants/100/photos')
+    .then((response) => {
```
Comment (cthao309) [inline client/src/components/Gallery.jsx:33]: Don't hard code the restaurant id, randomly generate an id from 1 to 100.

Comment (serviolee) [reply client/src/components/Gallery.jsx:33]: ok I can change it

File: database/Populator.js (hunk 1 above)
Comment (serviolee) [reply database/Populator.js:34]: Good point

**Pull request merged**